src/wrek.erl

```diff
@@ -239,23 +230,51 @@ make_vert_data(#state{dag = Dag}, Name) ->
     maps:from_list(Reaching).
 
 
--spec mark_vert_done(state(), pid()) -> state().
+-spec mark_vert(state(), pid(), done | failed) -> state().
 
-mark_vert_done(State = #state{children = Children, dag = Dag}, Pid) ->
+mark_vert(State = #state{children = Children, dag = Dag}, Pid, Status) ->
     #{Pid := Name} = Children,
     Children2 = maps:remove(Pid, Children),
     {Name, Label} = digraph:vertex(Dag, Name),
-    Label2 = Label#{done => true},
+    Label2 = Label#{done => Status},
```

Comment:
Isn't it better to replace `done` with `status`?

Reply:
Yes! In fact, I did it here: https://github.com/rkallos/wrek/pull/9/files#diff-b21bff10a440fd7dd2107422e92be096R237

Reply:
Yes, exactly :)